app/paysurvey/journey/ssj/SsjRequest.scala

```diff
@@ -33,9 +34,42 @@ final case class SsjRequest
       createdOn
     )
   }
-
 }
 
 object SsjRequest {
   implicit val format = Json.format[SsjRequest]
 }
+
+final case class SsjJourneyRequest
+  (
+    origin:         String,
+    returnMsg:      String,
```

Comment:
Do we need to accept a return message? Can we have generic template for survey?

Reply:
This way the caller can specific what the message is